amy/dashboard/views.py

```diff
@@ -81,10 +84,9 @@ def admin_dashboard(request):
     # assigned events that have unaccepted changes
     updated_metadata = Event.objects.active().filter(metadata_changed=True)
 
-    if assigned_to is not None:
-        current_events = current_events.filter(assigned_to=assigned_to)
-        unpublished_events = unpublished_events.filter(assigned_to=assigned_to)
-        updated_metadata = updated_metadata.filter(assigned_to=assigned_to)
```

Comment:
@lauryndbrown can you add tests for the scenario when user accesses `?assigned_to=`?

Reply:
@pbanaszkiewicz So just for clarification when the user visits the admin dashboard `/dashboard/admin/` no `assigned_to` is given. This is different from `assigned_to` being `None` which is `/dashboard/admin/?assigned_to=`. When it's `None` all events are shown. I should add the `if` statement back so that is more clear

Reply:
Yes, I understand. Actually for `?assigned_to=` you'll get only unassigned events, but this may be desired functionality for Sher.

Reply:
Ah so there is some  test coverage for `?assigned_to=` in` test_has_upcoming_events`. But let me add a test to ensure it handles all of those cases correctly

Reply:
That's great, thank you.